src/constant.js

```diff
@@ -30,10 +30,10 @@ export const PATHS = {
   VOLUNTEER_OVERVIEW: "/volunteer/:volunteerId",
   OUR_PARTNER: "/our-partner",
   OUR_STORY: "/our-story",
-  NEWUSER_DASHBOARED: "/user-dashboared",
-  PYTHONCOURSE: "/Python-course",
+  NEW_USER_DASHBOARED: "/user-dashboared",
+  PYTHON_COURSE: "/Python-course",
   SEARCHED_COURSE: "/search-course",
-  RETURNINGUSERPAGE: "/Returning-user",
+  RETURNING_USER_PAGE: "/Returning-user",
```

Comment:
Start with capital letters for `Python-course` and `/Returning-user` or lowercase to be consistent with our paths which are all in lowercase (in case a person enters it directly in the address bar)?